optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/planner/SolverIntegrationTest.java

```diff
@@ -74,7 +74,7 @@
 
     @BeforeEach
     void setUp() {
-        solverConfig = SolverConfig.createFromXmlResource(RouteOptimizerConfig.SOLVER_CONFIG);
+        solverConfig = SolverConfig.createFromXmlResource(Constants.SOLVER_CONFIG);
```

Comment:
Would it make sense to inject the SolverConfig into this test instead?

Reply:
Probably yes but I would have to turn the test into a `@SpringBootTest` that starts the whole Spring framework. I want to avoid that as much as possible.